app/controllers/obituaries_controller.rb

```diff
@@ -21,6 +21,27 @@ def show
     @obituary = Obituary.find(params[:id])
   end
 
+  def edit
+    @obituary = Obituary.find(params[:id])
+  end
+
+  def update
+    @obituary = Obituary.find(params[:id])
+    if @obituary.update(obituary_params)
+      flash[:success] = 'Obituary Updated!'
+      redirect_to obituary_path(@obituary.id)
+    else
+      flash[:error] = @obituary.errors.full_messages.to_sentence
+      render :edit
+    end
+  end
+
+  def destroy
+    Obituary.find(params[:id]).destroy
+    flash[:notice] = 'Obituary Deleted'
```

Comment:
I like that you added flash messages to all of these.

Reply:
upvote!